### PR TITLE
react1 week3 meal-sharing

### PR DESCRIPTION
**Pull request by slastushevskyi**

### CODE REVIEW

File: src/client/components/MealListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const MealListItem = ({ item }) => {
+  return (
```
Comment (shpomp) [inline src/client/components/MealListItem.jsx:4]: You could use object destructuring here and have 
`const {title, description, price} = item;
`Then you can use `title, description, price` directly 👍🏽

File: src/client/App.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
 function App() {
   return (
     <Router>
+      <Route exact path="/all-meals">
+        <MealListComponent />
+      </Route>
       <Route exact path="/">
-        <p>test</p>
+        <p>testa</p>
```
Comment (shpomp) [inline src/client/App.js:13]: 👀

**Review by shpomp — APPROVED** [review]
Perfect! ⭐

File: src/client/App.js (hunk 2 above)
Comment (slastushevskyi) [reply src/client/App.js:13]: This is how I tested frontend before writing main code 🤓

File: src/client/components/MealListItem.jsx (hunk 1 above)
Comment (slastushevskyi) [reply src/client/components/MealListItem.jsx:4]: Yes, I did that in todo homework.
Thank you for review, btw :)

**Pull request merged**